packages/vscode-docker-registries/src/clients/RegistryV2/registryV2Request.ts

```diff
@@ -52,7 +52,7 @@ async function registryV2RequestInternal<T>(options: RegistryV2RequestOptions):
 
     const request: RequestLike = {
         headers: {
-            accept: 'application/json',
+            accept: 'application/vnd.oci.image.index.v1+json',

```

Comment:
Since the `accept` header can have multiple options, this might be safer:

```suggestion
            accept: 'application/json,application/vnd.oci.image.index.v1+json',
```